src/components/Loading/Loading.tsx

```diff
@@ -0,0 +1,78 @@
+import styles from "./Loading.module.css";
+
+export default function Loading() {
```

Comment:
로딩 상황을 표시하는 컴포넌트에는 일반적으로 `Spinner`라는 이름을 많이 사용하는 것 같아요.

```suggestion
export default function Spinner() {
```

Reply:
컴포넌트가 가로 기준으로만 중앙 배치가 되는 것 같응데 세로 기준 중앙 배치는 컴포넌트를 사용하는 곳에서 해줘야 할까요?

Reply:
그런 의도로 구현을 했는데 달레님 코멘트 보니 가로세로 중앙정렬과 외부에서 className 주입해서 스타일 제어하도록 수정해야겠네요

Reply:
다시 생각해보니 spinner자체만 구현을 하고 사용하는 곳에서 위치를 제어하도록 수정하는게 나을거같네요!

Reply:
사용하는 곳에서 어떻게 spinner의 위치를 제어해야되는지를 보여주는 스토리를 추가해주시면 다른 분들이 나중에 사용하실 때 도움이 될 것 같습니다.

Reply:
오호..! 이부분은 decorators 사용하면 좋을거 같네요?
apporved됐지만 적용시킨다음에 merge하도록할게요!